frontend/app/form/jobdescriptionForm.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import { useState } from "react";
+import {
+	Button,
+	Card,
+	CardContent,
+	CardHeader,
+	TextField,
+} from "@mui/material";
+
+export default function JobDescriptionForm() {
+	const [description, setDescription] = useState<string>("");
+	const [message, setMessage] = useState<string | null>(null);
+
+	/**
+	 * Requires task 8 to submit form data
+	 * For now, I created a mock to submit and
+	 * return a response
+	 */
+	const uploadJobDescription = async (
```

Comment:
use `backendPost` from `app/util/fetching.ts` to make a post request, even if it will always fail for now.